Makefile

```diff
@@ -82,7 +82,9 @@ ENVTEST_K8S_VERSION ?= 1.24
 
 # Scorecard ImagePullPolicy is hardcoded to IfNotPresent
 # See: https://github.com/operator-framework/operator-sdk/pull/4762
-CUSTOM_SCORECARD_VERSION = $(IMAGE_VERSION)00001
+#
+# Suffix is the timestamp of the image build, compute with: date -u '+%Y%m%d%H%M%S'
+CUSTOM_SCORECARD_VERSION ?= 2.2.0-20220829183117
```

Comment:
Could this value be generated within `scorecard-build`? Nothing else seems to use that as a dependency, so I think it would only happen when the developer is intentionally building a new scorecard test image, and this way it isn't just a hardcoded value that needs manuak updating along with the rebuild.

I guess this would also mean the scorecard config patch needs to be updated with something like a `hack/scorecard-patch.yaml.in` too.

Reply:
How does this revision look? The version will be generated at build time, and the scorecard Kustomize patch will only get updated when the user runs `make scorecard-build`.

Reply:
Nice, I think that should work well enough.